spec.md

```diff
@@ -126,22 +120,19 @@ The "text" and "binary" designations refer to two flavors of encoding:
 - The *text* format is a platform-idiomatic map from (unicode) `string` to `string`
 - The *binary* format is an opaque byte array (and presumably more compact and efficient)
 
-Note that there is no expectation that different tracing systems Inject and Extract `Spans` in compatible ways. Though OpenTracing is agnostic about the tracing implementation, for successful inter-process handoff it's essential that the processes on both sides of a propagation use the same tracing implementation.
+Note that there is no expectation that different tracing systems Inject and Join `Spans` in compatible ways. Though OpenTracing is agnostic about the tracing implementation, for successful inter-process handoff it's essential that the processes on both sides of a propagation use the same tracing implementation.
 
 <div id="propagation-example"></div>
 
-#### An end-to-end Injector and Extractor propagation example
+#### An end-to-end Inject and Join propagation example
 
 To make the above more concrete, consider the following sequence:
 
-1. A client process has a `Span` instance and is about to make an RPC over a home-grown HTTP protocol
-1. That client process calls some `Tracer.Injector(...)` method, passing a format identifier for a string map
-1. With an `Injector` instance in hand, the client calls `injector.InjectSpan(...)`, passing the active `Span` instance and a string map carrier in as parameters
+1. A *client* process has a `Span` instance and is about to make an RPC over a home-grown HTTP protocol
+1. That client process calls some `Tracer.Inject(...)` method, passing the active `Span` instance, a format identifier for a string map, and a string map carrier as parameters
 1. The newly-populated (opaque) string map is encoded into the homegrown HTTP protocol (e.g., as headers) by the application code
 1. Now in the server process, the application code grabs the opaque string map from the homegrown HTTP protocol
```

Comment:
the term "opaque map" is quite vague here, could mean different things, i.e. it could be that the user knows it's a map but does not know what it contains, or that a user doesn't even know it's a map (because it is "encoded into headers").

Not sure if it can be expressed more clearly without overwhelming the reader with details better left for elsewhere.